examples/common/python/utility/utility.py

```diff
@@ -245,7 +255,25 @@ def decrypted_response(input_json, session_key, session_iv, data_key=None, data_
     return input_json_params['outData']
 
 #---------------------------------------------------------------------------------------------
+def verify_data_hash(msg, data_hash):
```

Comment:
For now it's fine to keep this function. But we need to revisit this utility.py and see how to organize it better by moving some functions outside this file as a separate PR. For example, encryption related utility functions can be move to encrypt_utils.py,  file related utility functiosn to file_utils.py and so on/